portal/src/texts/profile/picture.md

```diff
@@ -1,5 +1,288 @@
+
+Oversiktsbildene fra toppen av tinden gjør seg godt som korte filmer eller cinemagrafer (stillbilde med små repeterande bevegelser).
+
+Noter deg at stilen med bilder tatt i fotostudio med helsort eller helhvit bakgrunn er forbeholdt bilder i bevegelse.
+
+## Implementering
+
+Tenk gjennom dette før du velger et bilde:
+Passer bildet med Fremtinds visjon?
+Hjelper bildet brukeren å forstå det du skal kommunisere?
+Viser bildet produktet eller tjenesten på en god måte?
+Gir bildet innholdet verdi?
+Hva slags virkning har bildet på mottakeren?
+
+Bilder brukes i hovedsak som bakgrunn, i tekst eller i kort.
+
+Bilde som bakgrunn betyr at vi legger andre elementer oppå bildet, feks typografi eller kort. Bildene skal brukes på responsive nettsider, og både format og størrelse kan bli endret, samtidig som andre elementer kan endre plassering i forhold til bildet. Pass på at elementet som overlapper bildet ikke dekker over hovedmotivet.
+
+[animasjon: Bakgrunn på flere flater med elementer over. Feks Stein Olavs skjemaflyt (meld skade)]
+
+Som bakgrunn kan bildet være heldekkende. Vi bruker ikke heldekkende bilder uten designelementer, det ødelegger flyten til brukeren.
+
+Bilder i tekst fungerer som en hvilepause i teksten eller for å understreke eller forklare innholdet. Hvis bilde og tekst henger sammen er det viktig at bildet plasseres så nærme det relevante innholdet som mulig.
+
+Ved å bruke rikelig med luft rundt bildet konkurrerer det ikke med andre elementer og informasjon blir oppdelt og behagelig å lese.
+
+Bilder i tekst skal følge en todeling eller tredeling av gridet og forøvrig følge prinsippene for layout.
+
+[animasjon: Bilde som følger grid - vi ser gridet i bakgrunn]
+
+Velg 2-3 gode bilder heller enn mange. Forsøk å la kun 1-2 bilder være synlig i samme vindu.
+
+[animasjon: Scroll]
+
+Skru på gridfunksjon nederst i menyen for å se hvordan bildene følger gridet i Jøkul.
+
+Bilder i kort kan hjelpe leseren å navigere, men husk at for mange bilder kan virke mot sin hensikt og gjør det uoversiktlig. Bruk heller for eksempel typografi og bevegelse som virkemiddel hvis du har mer enn fem kort. Bruk bilder med et tydelig fokuspunkt og lite bakgrunnsstøy.
+
+[Bilde: Kort med rolig og uoversiktlig bilde + Eksempel på 4 og 8 kort]
+
+	•	Tenk gjennom bildets funksjon og virkning
+	•	Bilder med rom rundt motivet gir fleksibilitet
+	•	Bakgrunnsbilder kan være heldekkende
+	•	Plassér bildet nær relevant info
+	•	Bilder i tekst skal ha godt med luft rundt seg
+	•	Bildene skal følge prinsipper for layout
+	•	Bruk få, men gode bilder
+	•	Du kan bruke bilder i kort når du fem eller færre kort
+
+## Bildearkiv
+
+Fremtind har alle bruksrettigheter til alle bildene du finner i bildearkivet. Disse bildene er ferdig behandlet og tilpasset profilen.
+
+Trenger du et motiv som ikke finnes i arkivet kan du:
+- se om du finner noe i disse bildedatabasene med gratis bilder og fri bruk:
+www.unsplash.com
+...
+...
```

Comment:
Gratis bildebaser – noen av disse gir fri bruk som privatperson, men ikke for et helt selskap. Er det etisk forsvarlig å ikke betale for bildene som benyttes?
(Steinop)

Reply:
Vi har jo fått strenge retningslinjer for dette i Sopra Steria - vi kan ikke bruke nesten noe fra gratis baser. Sjekk med marked her om reglene er de samme.